app/scripts/modules/core/cluster/rollups.less

```diff
@@ -41,7 +41,7 @@ server-group {
   }
 }
 
-.rollup-entry {
+.row.rollup-entry {
```

Comment:
I made this selector more specific.  The CSS injected in the page seemed to have changed order, and the hither specificity for `.row` and `.rollup-entry` switched places. Any thoughts on why that happened?